blockchain/chain/src/store/chain_store.rs

```diff
@@ -664,16 +664,16 @@ where
     Ok(ts)
 }
 
-/// Helper to ensure consistent Cid -> db key translation.
+/// Helper to ensure consistent CID to db key translation.
```

Comment:
probably `db` should be `DB`, no strong feelings though

Reply:
Hmm.. I will keep `db` for now. I feel `db` is more an abbreviation for the word database. Plus it's also used [here](https://github.com/ChainSafe/forest/blame/main/scripts/sync_check/docker-compose.yml#L21) 😁